docs/source/create/entities/web-entity.rst

```diff
@@ -0,0 +1,130 @@
+##########################
+Play media in a Web Entity
+##########################
+
+Web entities can play a number of different media formats through embedded players.
+
+.. contents:: On This Page
+    :depth: 1
```

Comment:
The "On This Page" should exclude the page's title and include the section titles.

Reply:
I don't actually get what you mean with this

Reply:
Nevermind, working on it

Reply:
I cannot actually find a way to do this. I changed it to show the section titles. All the other pages in rst format also show the page title, so I would just leave it like this.